app/themes/colors.js

```diff
@@ -5,14 +5,16 @@
  * That allows to change them more easily later on.
  */
 
-const primary = '#000000';
-const text = '#212529';
+const primary = '#2F4858';
+const lightGreen = '#607274';
+const text = '#4D3D0C';
 const secondaryText = '#FFFFFF';
+const tertiaryText = '#4D3D0C';
 const secondary = '#b0b0b0';
 const success = '#28a745';
 const error = '#dc3545';
 const gotoStories = '#1890ff';
-const cardBg = '#d8d8d8';
+const cardBg = '#EFF1F3';
```

Comment:
Why are these changes there ?

Reply:
Current ui of app looked somewhat awful hence I tried to modify it. Have added screenshots in the PR description for the reference.